src/mepo3/cmdline/parser.py

```diff
@@ -88,18 +69,18 @@ def __clone(self):
             default = None,
             help = 'Branch/tag of URL to initially clone (Only allowed with URL!)')
         clone.add_argument(
-            '--config',
-            metavar = 'config-file',
+            '--registry',
+            metavar = 'registry',
             nargs = '?',
-            default = None,
-            help = 'Configuration file (ignored if init already called)')
+            default = 'components.yaml',
```

Comment:
Apologies if this has been handled, but I fear it may be hard to glean directly from staring at the code changes.   For hepo, we want the config file to be hidden in a subdirectory.   E.g., `.hepo/config.yml`.